examples/plot_ranking.py

```diff
@@ -0,0 +1,143 @@
+# Copyright (c) Fairlearn contributors.
+# Licensed under the MIT License.
+
+"""
+=========================================
+Ranking
+=========================================
+"""
+
+from fairlearn.metrics import exposure, utility, proportional_exposure
+from fairlearn.metrics import MetricFrame
+
+# %%
+# This notebook shows how to apply functionalities of :mod:`fairlearn.metrics` to ranking problems.
+# We showcase the example "Fairly Allocating Economic Opportunity" from the paper
+# `"Fairness of Exposure in Ranking" <https://dl.acm.org/doi/10.1145/3219819.3220088>`_
+# by Singh and Joachims (2018).
+# The example demonstrates how small differences in item relevance can lead to large differences
+# in exposure. Differences in exposure can be harmful not only because you are ranking individuals
+# directly but individuals can also be indirect affected by a ranking, think about books, music or
```

Comment:
Maybe first give a global explanation and add an example underneath this line for more clarity?
`For example, it could be that authors of color's products are consistently lower ranked in search results.`